.buildkite/pipeline.yml

```diff
@@ -0,0 +1,41 @@
+common-params:
+  &publish-android-artifacts-docker-container
+  docker#v3.8.0:
+    image: "public.ecr.aws/automattic/android-build-image:v1.1.0"
+    propagate-environment: true
+    environment:
+      # DO NOT MANUALLY SET THESE VALUES!
+      # They are passed from the Buildkite agent to the Docker container
+      - "AWS_ACCESS_KEY"
+      - "AWS_SECRET_KEY"
+
+steps:
+  - label: "Lint & Checkstyle"
+    key: "lint_and_checkstyle"
+    plugins:
+      - *publish-android-artifacts-docker-container
+    command: |
+      cp gradle.properties-example gradle.properties
+      ./gradlew lint checkstyle
+    artifact_paths:
+      - "**/build/reports/lint-results.*"
+      - "**/build/reports/checkstyle/checkstyle.*"
+  - label: "Test"
+    key: "test"
+    plugins:
+      - *publish-android-artifacts-docker-container
+    command: |
+      cp gradle.properties-example gradle.properties
+      ./gradlew test
```

Comment:
👍  for _not_ including the `--stacktrace` that we had in CircleCI before. Less verbose logs are easier to read, and having stacktrace by default on _every_ gradle commands is seldom useful, so I like these more compact logs.